src/neo/SmartContract/NefFile.cs

```diff
@@ -12,7 +13,7 @@ namespace Neo.SmartContract
     /// | Magic      | 4 bytes   | Magic header                                               |
     /// | Compiler   | 32 bytes  | Compiler used                                              |
     /// | Version    | 16 bytes  | Compiler version (Mayor, Minor, Build, Version)            |
-    /// | ScriptHash | 20 bytes  | ScriptHash for the script                                  |
+    /// | Abi        | Varbytes  | Abi file                                                   |
     /// +------------+-----------+------------------------------------------------------------+
     /// | Checksum   | 4 bytes   | First four bytes of double SHA256 hash                     |
```

Comment:
Do we need checksum anymore?

Reply:
I think that it's the same scenario, if we treat it as a file format, it's good to have a checksum, but not essential.

Reply:
Now we need to create the hash of the whole NEF file. So we don't need checksum.

Reply:
Done